### PR TITLE
INFRA: Initialize Build Project

### LINKED ISSUE #1: INFRA: Initialize Build Project
First issue for the Project ArtGallery.Services.

### PR DESCRIPTION
**Pull request by NaynaMallick**
Closes #1

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[*.cs]
+
```
Comment (ShriHumrudha) [inline .editorconfig]: remove extra line

File: ArtGallery.Services.Infrastructure.Build/Program.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+using ADotNet.Clients;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.SetupDotNetTaskV1s;
+using System.Collections.Generic;
+
+namespace ArtGallery.Services.Infrastructure.Build
+{
+    internal class Program
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Infrastructure.Build/Program.cs:13]: public class

File: ArtGallery.Services.Infrastructure.Build/Program.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using ADotNet.Clients;
```
Comment (ShriHumrudha) [inline ArtGallery.Services.Infrastructure.Build/Program.cs:5]: run code cleanup

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (NaynaMallick) [reply .editorconfig]: removed extra line

File: ArtGallery.Services.Infrastructure.Build/Program.cs (hunk 3 above)
Comment (NaynaMallick) [reply ArtGallery.Services.Infrastructure.Build/Program.cs:5]: applied code cleanup but no change found.

Comment (ShriHumrudha) [reply ArtGallery.Services.Infrastructure.Build/Program.cs:5]: image

Comment (NaynaMallick) [reply ArtGallery.Services.Infrastructure.Build/Program.cs:5]: yes, I did the same.

**Review by juturul — APPROVED** [review]

**Review by Shrabanac — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**